app/invocation/invocation_target_group_card.tsx

```diff
@@ -229,6 +229,13 @@ export default class TargetGroupCard extends React.Component<TargetGroupCardProp
                         <ChevronRight className="icon" />
                       </span>
                       <span className="target-label">{target.metadata?.label}</span>{" "}
+                      {target.testSummary && target.testSummary.totalNumCached > 0 && (
+                        <span className="target-cache-status">
+                          {target.testSummary.totalNumCached === target.testSummary.totalRunCount
+                            ? "Cached"
+                            : `${target.testSummary.totalNumCached} / ${target.testSummary.totalRunCount} runs cached`}
```

Comment:
Could run be confused with `--runs_per_test` here?

Reply:
I thought about "actions" or "spawns", but totally unsure what would give our users more clarity so I just stick with the `runs` from `total_run_count`... How about `1 / 3 total runs cached`?

Reply:
Not sure I'm grasping what these different fields mean, but is it correct that "total_num_cached" represents a "subset" of the attempts from "total_run_count"?

Also, I think it would be fine to either show "Cached" if it's 100% cached, or show nothing at all otherwise. Then when clicking through to the test summary page, we can show more detailed information about which runs were cached / which weren't?